admin_guide/seccomp.adoc

```diff
@@ -0,0 +1,149 @@
+[[admin-guide-seccomp]]
+= Restricting Application Capabilities Using Seccomp
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+[IMPORTANT]
+====
+Seccomp is
+link:https://github.com/kubernetes/kubernetes/blob/release-1.4/docs/design/seccomp.md[currently
+in alpha support].
```

Comment:
Alpha should be defined as https://github.com/openshift/origin#alpha-and-unsupported-kubernetes-features